### PR TITLE
chore: cleanups + use css + setup pre-commit hook

### PR DESCRIPTION
**Pull request by timurbazhirov**

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+echo ">>>>>>>> Husky Pre-commit: Transpiling sources"
+echo ""
+npm run transpile
+npm run transpile-tests
+git status
+
+echo ">>>>>>>> Husky Pre-commit: Adding transpiled sources in dist"
+echo ""
+git add dist
+echo ""
```
Comment (VsevolodX) [inline .husky/pre-commit]: End of file newline

File: dist/tests-legacy/cucumber/support/step_definitions/When I generate interpolated set with # intermediate images.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.default = _default;
+var _material_designer_page = require("../widgets/material_designer_page");
+function _default() {
+  this.When(/^I generate interpolated set with "([^"]*)" intermediate images$/, nImages => {
+    _material_designer_page.materialDesignerPage.designerWidget.generateInterpolatedSet(parseInt(nImages));
+  });
+}
```
Comment (VsevolodX) [inline dist/tests-legacy/cucumber/support/step_definitions/When I generate interpolated set with # intermediate images.js:12]: EOF as well

File: package.json [hunk 3]
```diff
@@ -5,15 +5,14 @@
...
         "build": "react-scripts build",
-        "prepublishOnly": "rm -rf dist; npm run transpile; npm run transpile-tests",
+        "prepare": "husky install",
         "transpile-tests": "babel --presets @babel/preset-env --out-dir dist/tests-legacy/cucumber tests-legacy/cucumber",
-        "transpile": "tsc; node-sass -r src/stylesheets -o dist/stylesheets",
-        "test": "bash run-tests.sh",
-        "testsOnly": "bash run-tests.sh -s=true",
+        "transpile": "tsc && npm run copy-css",
+        "copy-css": "cp src/stylesheets/* dist/stylesheets/",
+        "test": "echo 'Tests are in the tests subfolder. Use Node 20 (`nvm use 20`) and run `npm i` && `npm test` there'",
```
Comment (VsevolodX) [inline package.json:12]: Can we add a `cd tests && nvm use 20 && npm i && npm test && cd -` instead of a directive? Or is this too fragile?

**Review by VsevolodX — APPROVED** [review]
LGTM. MD works locally.

File: package.json (hunk 3 above)
Comment (timurbazhirov) [reply package.json:12]: Good idea, but with nvm there's too much logic in this command. Let's wait until we are on Node 20 for both app and tests

File: dist/tests-legacy/cucumber/support/step_definitions/When I generate interpolated set with # intermediate images.js (hunk 2 above)
Comment (timurbazhirov) [reply dist/tests-legacy/cucumber/support/step_definitions/When I generate interpolated set with # intermediate images.js:12]: This is dist

**Pull request merged**